client/src/components/pages/HomePage/HomeText.tsx

```diff
@@ -19,7 +19,7 @@ const HomeText = () => {
         <Link href="https://allenai.org/" target="_blank" rel="noopener noreferrer">
           Allen Institute for AI
         </Link>{' '}
-        (data release of May 26, 2020), which currently contains over 47,000 scholarly articles,
+        (data release of June 12, 2020), which currently contains over 47,000 scholarly articles,
```

Comment:
it has more than 47k, right?

Reply:
website says 50k, will update

Reply:
in the description here: https://www.semanticscholar.org/cord19/download

Reply:
I think it is ~120k (with ~60k full texts), the website must be outdated...

Reply:
How about I just remove this sentence, I don't think it really adds much

Reply:
Let's go by: https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/2020-06-12/changelog

```
---SUMMARY---
total metadata rows: 140780
CORD UIDs (new: 280, removed: 1)

Full text:
  PDF - 72027 json (new: 258, removed: 1943)
  PMC - 52614 json (new: 270, removed: 1)
```

I'd just say 140k articles. I think it's still important to give the user a sense of the size of the corpus.

Reply:
updated to 140k